test/functional/api/v2/compute_resources_controller_test.rb

```diff
@@ -148,6 +150,43 @@ def teardown
     assert !available_storage_domains.empty?
   end
 
+  test "should get available availability flavors" do
+    flavor = Object.new
```

Comment:
Feel free to use `Fog.mock!` instead of stubbing. That will return sample objects for all Fog calls such as  `.flavors`, `.security_groups`, etc... It's faster too. So in the end it'd look somewhat like this:

```ruby
context 'ec2' do # There's probably a better name..
  setup       { Fog.mock! }
  teardown { Fog.unmock! }
  test ...
  test ...
end
```